ConcernsCaseWork/ConcernsCaseWork/Pages/Shared/_Layout.cshtml

```diff
@@ -20,25 +20,29 @@
             })(window,document,'script','dataLayer','GTM-NGC8DRV');
         </script>
         <!-- End Google Tag Manager -->
-    }
 
-    <!-- Hotjar Tracking Code for https://amsd-casework-staging.london.cloudapps.digital/ -->
-    <script nonce="@nonce">
-        (function(h,o,t,j,a,r){
-            h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
-            h._hjSettings={hjid:3090373,hjsv:6};
-            a=o.getElementsByTagName('head')[0];
-            r=o.createElement('script');r.async=1;
-            r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
-            a.appendChild(r);
-        })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
-    </script>
+
+        <!-- Hotjar Tracking Code for https://amsd-casework.london.cloudapps.digital -->
+        <script nonce="@nonce">
+            (function (h, o, t, j, a, r) {
+                h.hj = h.hj || function () { (h.hj.q = h.hj.q || []).push(arguments) };
+                h._hjSettings = { hjid: 3090373, hjsv: 6 };
+                a = o.getElementsByTagName('head')[0];
+                r = o.createElement('script'); r.async = 1;
+                r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
+                a.appendChild(r);
+            })(window, document, 'https://static.hotjar.com/c/hotjar-', '.js?sv=');
+        </script>
+    }
 
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
     <meta name="theme-color" content="#0b0c0c">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta property="og:image" content="~/assets/images/govuk-opengraph-image.png">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self'; script-src 'self' https://static.hotjar.com https://www.googletagmanager.com/gtm.js 'nonce-@nonce';">
```

Comment:
Do we need the style-src setting?

Reply:
I found this on the hotjar site

Reply:
If we are not using any external styles then we don't need it, however removing it from the CSP would mean there will be no restrictions.
Thanks for the link, interesting that they have added individual urls then at the end putting 'unsafe-inline', which doesn't make sense as this will make the whole thing to ignore the restrictions and allow any scripts!
But I think using their wildcard urls would be useful!

Reply:
I *think* that setting will block inline styles as well as external ones, according to this: https://content-security-policy.com/style-src/ . Hopefully we aren't using any inline styles though so it shouldn't be an issue?

Reply:
I see, in terms of styles seem to be fine locally! Although I still see the CSP error when running locally! Trying to fix it locally before merging it!